processor/stash/stash.go

```diff
@@ -184,60 +186,66 @@ func (st *HandleT) storeErrorsToObjectStorage(jobs []*jobsdb.JobT) (errorJob []E
 			continue
 		}
 		path := fmt.Sprintf("%v%v.json.gz", tmpDirPath+localTmpDirName, fmt.Sprintf("%v.%v.%v.%v.%v", time.Now().Unix(), config.GetString("INSTANCE_ID", "1"), fmt.Sprintf("%v-%v", jobs[0].JobID, jobs[len(jobs)-1].JobID), uuid, workspaceID))
-		dumps[workspaceID] = path
+		if err := os.MkdirAll(filepath.Dir(path), os.ModePerm); err != nil {
+			panic(fmt.Errorf("creating gz file %q: mkdir error: %w", path, err))
+		}
+		writer, err := misc.CreateGZ(path)
+		if err != nil {
+			panic(err)
+		}
+		writerMap[workspaceID] = writer
```

Comment:
We only need to store the writer to get the filename, so does it make sense to keep a list of filenames instead?